### ISSUE TITLE
Empty transcript after starting

### ISSUE THREAD

**Issue by pberman-aws** [aws-samples/automated-meeting-scribe-and-summarizer#15]
In some meetings even after starting, once the meeting ends I get an empty transcript and an email with just attendees and nothing else.

**Comment by lawtonpittenger** [comment]
Hello,
Have you requested access to the Claude 3 Sonnet model on Amazon Bedrock? You'll need to read and accept the end-user license agreements or EULA to allow the transcriptions this solution to work properly.

**Comment by pberman-aws** [comment]
Yes, I have access to all the models. I should have added that I have the tool working properly in most cases and receive transcripts, this is more of an exception case than a setup issue.
